src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let fixedRobot = {};
+
+  for (const key in robot) {
+    if (fixedRobot.hasOwnProperty(robot[key])) {
+      fixedRobot = null;
```

Comment:
no need to assign null to robot - just return null from function

Reply:
done